Vladislav Pozniak/0/run.rb

```diff
@@ -0,0 +1,79 @@
+require 'roo'
+require 'roo-xls'
+
+# Level 1
+current = Dir.glob('./data/*').max
+table = Roo::Spreadsheet.open(current, extension: File.extname(current))
+
+print "What price are you looking for?\n>\s"
+item = gets.chomp
+
+found = 0
+products = []
+prices = []
+(9..table.last_row).each do |row|
+  next unless table.cell(row, 'A').to_s.start_with? item.upcase
+
+  products << product = table.cell(row, 'A')
+  prices << price = table.cell(row, 7)
+  puts "'#{product.capitalize}' is #{price} BYN in Minsk these days."
+  found += 1
+end
+
+if found.zero?
+  puts "'#{item}' can not be found in database."
+  exit
+end
+
+# Level 2
+prices_min = [].replace(prices)
+prices_max = [].replace(prices)
+month_min = File.basename(current, File.extname(current))
+month_max = File.basename(current, File.extname(current))
+(0...products.length).each do |prod|
+  Dir.glob('./data/*').each do |file|
+    sheet = Roo::Spreadsheet.open(file, extension: File.extname(file))
```

Comment:
I hate loading 118 files for checking few rows

Reply:
Shit happens :)